content/tutorials/demo-ably-tower-defense.textile

```diff
@@ -0,0 +1,40 @@
+---
+ably_product: reactor
+platform: browser
```

Comment:
This isn't really designed for browser, I've just made a hack which can eventually deploy to browser. Maybe go for `mixed` ?

Reply:
(same as above) ignore

Reply:
changed to mixed anyway